### ISSUE TITLE
HtmlEditor - Resize Media Blocks

### ISSUE THREAD

**Issue by KsyushaM** [DevExpress/DevExtreme#7269]
# The Problem

The HtmlEditor does not allow users to resize embedded media blocks.

# The Proposed Solution
We added a new `resizing` option that accepts an object with the following structure:

```js
mediaResizing: {
  enabled: true, // enables resizing
  allowedTargets: ["images"] // specifies resizable media types
}
```

## UI Design
![resizing_example](https://user-images.githubusercontent.com/9039982/54197655-8ca72d80-44d5-11e9-95f4-725cea9556b8.png)

# Try It
## Live Sandbox

[jQuery](https://codepen.io/Doka/pen/wOrOoQ?&editable=true) 

## Installation
Coming soon

# We Need Your Feedback
## Take a Quick Poll

[Do you find resizing in the HtmlEditor useful?](https://goo.gl/forms/wEPbZx9fjlQvMlM93)

# Get Notified of Updates
Subscribe to this thread or to our [Facebook](https://www.facebook.com/DevExpress.DevExtreme/) and [Twitter](https://twitter.com/devextreme) accounts for updates on this topic.

**Comment by stenet** [comment]
This feature is great but when resizing an image it should keep aspect ratio.

**Comment by DokaRus** [comment]
Thanks @stenet !
We're discussing it with the team and definitely add it later.

**Comment by dxbykov** [comment]
Thank you to everybody who gave us feedback on this feature. It's available in the [v19.1](https://github.com/DevExpress/DevExtreme/releases/tag/19.1.3) release. I'm closing this thread. In the case of bugs or questions, feel free to create new GitHub [issues](https://github.com/DevExpress/DevExtreme/issues) or tickets in our [Support Center](https://www.devexpress.com/Support/Center/).
